Clean/Tables/Fibonacci32.lean

```diff
@@ -323,17 +313,15 @@ def formal_fib32_table : FormalTable (F p) RowType := {
       simp only [U32.value, fib32]
       rw [b0, b1, b2, b3, b4, b5, b6, b7]
       simp [ZMod.val_one]
-      simp only [U32.is_normalized, b0, b1, b2, b3]
-      simp only [ZMod.val_zero, Nat.ofNat_pos, and_self]
+      simp only [U32.is_normalized, b0, b1, b2, b3, b4, b5, b6, b7]
+      simp only [ZMod.val_zero, ZMod.val_one, Nat.ofNat_pos, and_self]
+      trivial
 
     -- inductive step
-    · intro lookup_h
-      simp only [TraceOfLength.forAllRowsOfTrace.inner, Fin.isValue] at lookup_h
-
-      -- first of all, we prove the inductive part of the spec
+    · -- first of all, we prove the inductive part of the spec
       unfold TraceOfLength.forAllRowsOfTraceWithIndex.inner
       intros constraints_hold
-      specialize ih2 lookup_h.right
+      -- specialize ih2 lookup_h.right
```

Comment:
`specialize ih2 lookup_h.right` can be removed instead of commented out, right?

Reply:
right, appreciate the diligence!